buildenv/docker/mkdocker.sh

```diff
@@ -636,14 +636,15 @@ create_git_cache() {
   add_git_remote jdk8    https://github.com/ibmruntimes/openj9-openjdk-jdk8.git
   add_git_remote jdk11   https://github.com/ibmruntimes/openj9-openjdk-jdk11.git
   add_git_remote jdk14   https://github.com/ibmruntimes/openj9-openjdk-jdk14.git
+  add_git_remote jdk15   https://github.com/ibmruntimes/openj9-openjdk-jdk15.git
   add_git_remote jdknext https://github.com/ibmruntimes/openj9-openjdk-jdk.git
   add_git_remote omr     https://github.com/eclipse/openj9-omr.git
   add_git_remote openj9  https://github.com/eclipse/openj9.git
   echo " && echo Fetching repository cache... \\"
-  echo " && git fetch jdk14 \\"
+  echo " && git fetch jdk15 \\"
   echo " && git fetch --all \\"
   echo " && echo Shrinking repository cache... \\"
-  echo " && git gc --aggressive --prune=all"
+  echo " && git repack -a -d -f --depth=50 --window=250 --threads=1"
```

Comment:
For the record, what is the explanation for using `--window=250`, when the default is 10?

Reply:
The suggestion is attributed to Linus Torvalds [here](https://stackoverflow.com/questions/28720151/git-gc-aggressive-vs-git-repack/).

The git gc [documentation](https://git-scm.com/docs/git-gc) explains that `--depth=50` and `--window=250` are the defaults when `--aggressive` is used.